src/app/origin-destination/components/origin-destination.component.html

```diff
@@ -1,2 +1,33 @@
-<div id="main-origin-destination-container-root" style="width: 98%; height: 100%; position: relative;">
-</div>
\ No newline at end of file
+<div class="matrix-wrapper">
+  <div
+    id="main-origin-destination-container-root"
+    style="width: 98%; height: 100%"
+  ></div>
+
+  <div class="palette-buttons">
+    <div
+      class="palette-option custom"
+      [class.active]="colorSetName === 'custom'"
+      title="Custom palette"
+      (click)="onChangePalette('custom')"
+    ></div>
+    <div
+      class="palette-option blue"
+      [class.active]="colorSetName === 'blue'"
+      title="Blue palette"
+      (click)="onChangePalette('blue')"
+    ></div>
+    <div
+      class="palette-option orange"
+      [class.active]="colorSetName === 'orange'"
+      title="Orange palette"
+      (click)="onChangePalette('orange')"
+    ></div>
+    <div
+      class="palette-option gray"
+      [class.active]="colorSetName === 'gray'"
+      title="Gray alette"
```

Comment:
```suggestion
      title="Gray palette"
```

Nit: we could also translate it